packages/itwin/grouping-mapping-widget/src/widget/components/MappingImportWizardModal.scss

```diff
@@ -9,9 +9,18 @@
   flex-direction: column;
   gap: $iui-baseline * 2;
   margin-top: $iui-baseline * 2;
+  margin-right: $iui-baseline * 2;
+  margin-left: $iui-baseline * 2;
```

Comment:
The modal comes with the necessary padding. Tbh, I think we should just remove the margins from the container.

Reply:
Yeah, I was on the fence about it, but I'm fine removing it.
I was trying to maintain the spacing that existed before, but it isn't necessary.